### PR TITLE
Minor docs update

### PR DESCRIPTION
**Pull request by steshaw**
- a few minor corrections
- Update to use the helper script for running the frontend

### CODE REVIEW

**Review by simeoncarstens — APPROVED** [review]
Thanks a lot! I wonder if all those lower-case capitalization were not the result of a careless search-and-replace at some point :thinking:

File: docs/deployment.md
Comment (simeoncarstens) [inline docs/deployment.md:1]: Looks like I approved prematurely, as there's a merge conflict now :slightly_frowning_face:

Comment (steshaw) [reply docs/deployment.md:1]: I'll take a look at that

**Comment by steshaw** [comment]
I'm not sure what happened there. I didn't intend to close the PR 🤷‍♂️. It seems that it has been merged already.

**Comment by Niols** [comment]
> @steshaw wants to merge 0 commits into `main`

Looks like it indeed!

**Pull request closed without merging**